pkg/occlient/namespace.go

```diff
@@ -0,0 +1,92 @@
+package occlient
+
+import (
+	"time"
+
+	"github.com/pkg/errors"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/fields"
+	"k8s.io/apimachinery/pkg/watch"
+	"k8s.io/klog"
+)
+
+// CreateNamespace creates new namespace
+func (c *Client) CreateNamespace(name string) (*corev1.Namespace, error) {
```

Comment:
I think the namespace functions related things should we moved to the kClient package as we have been adding the kubernetes related things like ingress etc there.

Reply:
problem is that when I do that I will have to add pass whole `kclient` to all function that deal with project :-/

Reply:
How about instead of passing the whole occlient or the kclient, we pass the context? The devfile context can contain both the clients and we can use them accordingly.

Reply:
I can do that. But wouldn't it be weird if we use a struct that is from `genericclioptions` for all project functions?

Reply:
I've tried that in the latest commit (a18c72cb03a00ca548feac60de287d148b56a3f4) take a look

Reply:
I agree it would be weird to use this struct, but it might create confusion if we keep the namespace parts in occlient. In future we can try to find a way to make a common abstracted client.

Reply:
So what do you suggest, should I remove the last commit and keep functions in `occlient` or should I use context? I have both options ready :-)

Reply:
Let's keep it here in kclient package for now.